### PR TITLE
[@types/feathersjs__errors] Added typings for the missing "code", "className" and "data" property members of the FeathersError object.

### PR DESCRIPTION
**Pull request by RazzM13**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [please see the listing of the FeathersError fields](https://docs.feathersjs.com/api/errors.html#feathers-errors)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/feathersjs__errors/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
 // Type definitions for @feathersjs/errors 3.2
 // Project: http://feathersjs.com/
 // Definitions by: Jan Lohage <https://github.com/j2L4e>
+//                 RazzM13 <https://github.com/RazzM13>
 // Definitions: https://github.com/feathersjs-ecosystem/feathers-typescript
 // TypeScript Version: 2.2
 
 export class FeathersError extends Error {
-    constructor(msg: string | Error, name: string, code: number, className: string, data: any)
+    code: number;
```
Comment (PranavSenthilnathan) [inline types/feathersjs__errors/index.d.ts]: These should probably be readonly.

File: types/feathersjs__errors/index.d.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
 // Type definitions for @feathersjs/errors 3.2
```
Comment (PranavSenthilnathan) [inline types/feathersjs__errors/index.d.ts]: Bump version to 3.3

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

File: types/feathersjs__errors/index.d.ts (hunk 1 above)
Comment (RazzM13) [reply types/feathersjs__errors/index.d.ts]: I've marked all `FeathersError` specific properties as read-only, hopefully this is OK.

File: types/feathersjs__errors/index.d.ts (hunk 2 above)
Comment (RazzM13) [reply types/feathersjs__errors/index.d.ts]: Done.

**Comment by RazzM13** [comment]
Hello @PranavSenthilnathan, I've also added the missing `toJSON` function; unfortunately I missed this last time because it is just vaguely referenced in their documentation [here](https://docs.feathersjs.com/api/errors.html#feathers-errors) but is clearly visible in the [code](https://github.com/feathersjs/feathers/blob/master/packages/errors/lib/index.js#L69-L82).

Thank you for your support and sorry for the delay on my end.

**Review by PranavSenthilnathan — APPROVED** [review]

**Comment by PranavSenthilnathan** [comment]
Congratulations on your first DefinitelyTyped contribution! Thank you for being a part of the community!

**Pull request merged**